### ISSUE TITLE
Corrupted text in .smv file

### ISSUE THREAD

**Issue by Er9y714** [firemodels/fds#5883]
When I run this simulation I cannot execute the .smv file. In text editor, there seems to be foreign characters.
![image](https://user-images.githubusercontent.com/26166281/34876438-d3f239e6-f7a1-11e7-964b-39c212ef9780.png)
For running please change the extension of the second file (uvw file) as well, into .csv.
[Test.fds.txt](https://github.com/firemodels/fds/files/1626064/Test.fds.txt)
[1660m_v4_uvw_t001_m001.csv.txt](https://github.com/firemodels/fds/files/1626065/1660m_v4_uvw_t001_m001.csv.txt)

Could you please have a look?

Also I have an ongoing calculation that has this problem in .smv file. If I remove this line (seen below) can I trust the rest of the smv file to work properly?

Thank you.

**Comment by gforney** [comment]
1.  copy your .smv file to another file then remove the line (don't want fds to be outputting to the .smv file while you are editing it).  you should be able to view the case without a problem
2.  I am not able to reproduce your problem.  When I run Test.fds I can view it in smokeview without a problem.  Is Test.fds the case that was causing you problems?

**Comment by Er9y714** [comment]
1. Thank you. It works. 
2. Yes, I reproduced it in FDS 6.6.0 and SMV 6.6.0 also tried in two PCs. One is Windows 10, other Win 7.

**Comment by gforney** [comment]
I renamed test.fds.txt to test.fds and ran it using fds 6.6.0 on a mac, windows and linux system.  It works for me on all three platforms.  Are you doing something different than what I did?

![test_0080](https://user-images.githubusercontent.com/12403014/34878811-048fc326-f779-11e7-99b8-48646037be03.png)

**Comment by Er9y714** [comment]
Just to make sure, did you also include 1660m_v4_uvw_t001_m001.csv?

**Comment by gforney** [comment]
I do not see any reference in test.fds to this file

**Comment by Er9y714** [comment]
```
  

&HEAD CHID='1660m_v4', TITLE='A'/
&TIME  T_BEGIN=150 T_END=1800.0/
&DUMP   DT_BNDF=10.0, DT_DEVC=5.0, DT_PL3D=60.0, DT_RESTART=30.0, DT_SLCF=5.0, /
 
&MESH ID='05 04 02-a-a', IJK=92,24,34, XB=1602.0,1648.0,0.0,9.6,0.0,6.8/

&CSVF UVWFILE='1660m_v4_uvw_t001_m001.csv'/ 
&SLCF QUANTITY='VELOCITY', PBY=5.2/
   

&TAIL /

```
See the line with &CSVF. Am I wrong ?

**Comment by gforney** [comment]
a windows glitch on my end.  I have the right file now

**Comment by gforney** [comment]
try to reproduce the problem with a shorter run time.  don't have time today to wait for the case to run out to 1800 s

**Comment by Er9y714** [comment]
1800 is arbitrary. I had the problem starting from the start (as soon as .smv is generated)

**Comment by gforney** [comment]
kevin - the following lines in dump.f90 are outputting bogus non-ascii characters to the .smv file .  I'll look at smokeview to get it to ignore them without crashing (by ignoring the entry) - could you look at it on the fds side

The file name in CSVFINFO(N)%CSVFILE is corrupted for some reason.  These lines in dump.f90 are what are outputting the bad characters to the .smv file:

DO N = 1, N_CSVF
  WRITE(LU_SMV,'(/A)') 'CSVF'
  WRITE(LU_SMV,'(1X,A)') 'ext'
  WRITE(LU_SMV,'(1X,A)') TRIM(CSVFINFO(N)%CSVFILE)
END DO

**Comment by mcgratta** [comment]
OK, I'll look.

**Comment by gforney** [comment]
should be fixed in 56d94414 .  

I'll look at modifying smokeview so it can handle the bad entry. 

In the mean time, if you can't build fds, you'll have to replace the bogus characters in the CSVFILE entry by hand with some ascii text such as null

**Comment by Er9y714** [comment]
Thank you for looking into it. 
I can access the smv file by removing these lines, as you suggested earlier. 

If it is solved, should I close the issue?
